examples/private_set_intersection/module/rust/Cargo.toml

```diff
@@ -7,9 +7,9 @@ authors = ["Tiziano Santoro <tzn@google.com>"]
 crate-type = ["cdylib"]
 
 [dependencies]
-oak = { path = "../../rust/oak" }
-oak_derive = { path = "../../rust/oak_derive" }
+oak = { path = "../../../../rust/oak" }
+oak_derive = { path = "../../../../rust/oak_derive" }
```

Comment:
This is not particularly nice, I wonder if we should do something about it (move things around, symlinks, something else..)

Reply:
Agreed. I was planning to look at cargo a bit more to see if we can have a root folder defined.